### PR TITLE
Revamp UI and fixed Code Climate duplicate code

### PR DESCRIPTION
**Pull request by MinhDang98**

### CODE REVIEW

File: zlp-scheduler/db/schema.rb [hunk 1]
```diff
@@ -15,6 +15,16 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "adminstudrecs", force: :cascade do |t|
```
Comment (BWitemeyer) [inline zlp-scheduler/db/schema.rb:18]: I think that Austin's PR eliminated this. Could you check that you have the changes to the migrations pulled to your fork?

**Review by BWitemeyer — CHANGES_REQUESTED** [review]
The UI all looks great, I just have a couple comments on other files that were changed.

File: zlp-scheduler/app/controllers/users_controller.rb [hunk 2]
```diff
@@ -99,15 +99,15 @@ def update_user
...
-          redirect_to view_term_admin_path, :notice => "Logged in !" 
-        else
-          if DateTime.current >= @term.opendate && DateTime.current < @term.closedate
-            redirect_to '/student/view_terms', :notice => "Logged in !" 
-          else
-            redirect_to '/student/closed', :notice => "Logged in !" 
-          end 
-        end
+        # if current_user.admin?
+        #   redirect_to view_term_admin_path, :notice => "Logged in !"
```
[the remark this answers is not in the document]
Comment (MinhDang98) [reply zlp-scheduler/app/controllers/users_controller.rb:103]: Code Climate tells me that this is a duplicate code section in both users_contrller.rb and session_controller.rb. I had tried to comment out the session_controller code but it stops working and I tried to comment out this code and it working.

File: zlp-scheduler/db/schema.rb (hunk 1 above)
Comment (MinhDang98) [reply zlp-scheduler/db/schema.rb:18]: That's weird because the last time I check my fork is up to date with the master.

Comment (adauzat90) [reply zlp-scheduler/db/schema.rb:18]: When you ran the migrations did you completely drop the tables and run the migrations?

File: zlp-scheduler/app/controllers/users_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (BWitemeyer) [reply zlp-scheduler/app/controllers/users_controller.rb:103]: Ok. I guess that's fine, then.

**Pull request merged**